### DISCUSSION TITLE
What's the best way to create a cell by gene dataframe?

### DISCUSSION THREAD

**Discussion by wshands** [GreenleafLab/ArchR#1069] [Questions / Documentation]
I'm a bit unfamiliar with R and ArchR and I'd like to create a data frame with rows that match a cell name to gene information, e.g.
<cell name>  seqnames     start       end  strand        (gene)name     idx

Where myGeneScoreMatrix <- getMatrixFromProject(proj) and the cell name from colData(myGeneScoreMatrix) and the gene information from rowData(myGeneScoreMatrix). What's the best way to do that?

Thanks!

**Comment by rcorces** [comment]
I'm not sure I understand your question but it sounds like you just want to manipulate SummarizedExperiments and I'd recommend googling around for your answer.
